layer/clients/executor_service.py

```diff
@@ -63,6 +65,12 @@ def get_or_create_cluster(
             idle_scale_down_duration=idle_scale_down_duration,
             idle_shut_down_duration=idle_shut_down_duration,
             max_number_of_workers=10,  # TODO(emin): remove this parameter
-            worker_fabric="",  # TODO(emin): remove this parameter as it should be defined by the function executable for each
+            worker_fabric="f-small",  # TODO(emin): remove this parameter as it should be defined by the function executable for each
+            account_id=AccountId(value=str(account_id)),
```

Comment:
nit: we can use an "account_proto_mapper" here instead of creating the AccountId directly

Reply:
Will address on the coming ones as there are TODOs as well